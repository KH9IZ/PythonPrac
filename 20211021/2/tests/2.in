:fr f range(f)
:br b print(b)
:george g g**2+g
fr 100
br "QQ"
george 100500
quit QKRQ

