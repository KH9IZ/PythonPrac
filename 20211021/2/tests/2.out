range(0, 100)
QQ
None
10100350500
4
